exec/coroparse.c

```diff
@@ -1264,6 +1360,17 @@ static int main_config_parser_cb(const char *path,
 	    key, min_val, max_val, value);
 	*error_string = formated_err;
 
+	return (0);
+
+icmap_set_error:
+	if (snprintf(formated_err, sizeof(formated_err),
+	    "Can't store key \"%s\" into icmap, returned error is %s",
+	    key, cs_strerror(cs_err)) >= sizeof(formated_err)) {
```

Comment:
I don't have strong opinions here but using key_name rather than key might be more informative as it shows the full path that's trying to be stored.

Also, when parsing a file that's just 'a:a' I get the error
parser error: /etc/corosync/corosync.conf:1: Can't store key " into icmap, returned error is CS_ERR_NAME_TOO_LONG 
the string contains just ^H (hence the single quote mark) for some reason. Not sure why that's 'too long' though!

Reply:
Thank you for the review.

1. Added patch
2. How were you able to achieve that ^H? I mean, I've retried with files where 'a:a' was with newline, without newline (and for sure also with apostrophe :) ), with and without valgrind (hoping it will show error) on COS7 and next rhel and still unable to reproduce that issue.
3. I understand why it is CS_ERR_NAME_TOO_LONG. icmap function `icmap_check_key_name` checks length of key_name and validity of the name. I do not remember exactly why I've choose ERR_NAME_TOO_LONG, but it's probably because I was unable to find better CS_ERR.